exporters/trace/zipkin/zipkin.go

```diff
@@ -0,0 +1,85 @@
+package zipkin
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+)
+
+type Exporter struct {
+	url    string
+	client *http.Client
+	logger *log.Logger
+}
+
+var (
+	_ export.SpanBatcher = &Exporter{}
+)
+
+type Options struct {
+	Client *http.Client
+	Logger *log.Logger
+}
+
+type Option func(*Options)
+
+func WithLogger(logger *log.Logger) Option {
+	return func(opts *Options) {
+		opts.Logger = logger
+	}
+}
+
+func WithClient(client *http.Client) Option {
+	return func(opts *Options) {
+		opts.Client = client
+	}
+}
+
+func NewExporter(url string, os ...Option) (*Exporter, error) {
```

Comment:
Making `url` a [URL](https://golang.org/pkg/net/url/#URL) would provide a bit stronger guarantees around correctness. That way it isn't all the way to when the first Span is sent that it is figured out it is a malformed URL.

Reply:
How about I validate that inside the `NewExporter` function? I haven't seen `url.URL` used anywhere as a parameter - we usually use strings.

Reply:
Went with the validation inside the function for now.